src/gluonts/torch/model/deepar/_lightning_module.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytorch_lightning as pl
+import torch
+
+from gluonts.torch.modules.loss import DistributionLoss, NegativeLogLikelihood
+from gluonts.torch.util import weighted_average
+
+from ._module import DeepARModel
+
+
+class DeepARLightningModule(pl.LightningModule):
+    def __init__(
+        self,
+        model: DeepARModel,
+        loss: DistributionLoss = NegativeLogLikelihood(),
+        lr: float = 1e-3,
+        weight_decay: float = 1e-8,
+    ) -> None:
+        super().__init__()
+        self.model = model
+        self.loss = loss
+        self.lr = lr
+        self.weight_decay = weight_decay
+
+    def _compute_loss(self, batch):
+        params, scale, _, _ = self.model.unroll_lagged_rnn(
+            batch["feat_static_cat"],
+            batch["feat_static_real"],
+            batch["past_time_feat"],
+            batch["past_target"],
+            batch["past_observed_values"],
+            batch["future_time_feat"],
+            batch["future_target"],
+        )
+        distr = self.model.output_distribution(params, scale)
+        past_target = batch["past_target"]
+        future_target = batch["future_target"]
+        past_observed_values = batch["past_observed_values"]
+        future_observed_values = batch["future_observed_values"]
```

Comment:
Put these assignments at the top? They can already be used in `unroll_lagged_rnn`